### PR TITLE
[Example] Add Intelligent Collaboration (Video Conference)

### LINKED ISSUE #1007: Video conference application [CV]
WebNN semantic segmentation use case enhancement, it's OK to create a new example rather than stick to [semantic segmentation example](https://intel.github.io/webml-polyfill/examples/semantic_segmentation/).

Following Intelligent Collaboration features or APIs are needed:

Precondition features
- [x] Host: WebRTC server setup - Open WebRTC Toolkit (OWT)
- [x] Client: Rest API server enabling
- [x] Video Conference enabling with multiple attendees

Web Machine Learning features
- [x] Performance indicators
- [x] Blur my background (Semantic Segmentation)
- [x] Replace my background (Semantic Segmentation)
- [x] Public SS to WebRTC server

Comprehensive features
 - [x] UI and layout for desktop
 - [x] More conference features - Video pause
 - [x] More conference features - Mute
 - [ ] More conference features - Audio only
 - [x] More conference features - Screen Sharing
 - [x] More conference features - Conversation / Text message...
 - [x] UE - Full screen
 - [ ] UE - Seamless switching between normal video and inferenced canvas to WebRTC server
 - [ ] UI and layout for mobile
 - [ ] Performance and optimization

### PR DESCRIPTION
**Pull request by ibelem**
Fix #1007 with following features:

Precondition features
- [x] Host: WebRTC server setup - Open WebRTC Toolkit (OWT)
- [x] Client: Rest API server enabling
- [x] Video Conference enabling with multiple attendees

Web Machine Learning features
- [x] Performance indicators
- [x] Blur my background (Semantic Segmentation)
- [x] Replace my background (Semantic Segmentation)
- [x] Public SS to WebRTC server

Comprehensive features
 - [x] UI and layout for desktop
 - [x] More conference features - Video pause
 - [x] More conference features - Mute
 - [x] More conference features - Screen Sharing
 - [x] More conference features - Conversation / Text message...
 - [x] UE - Full screen

Image copyrights:

- Most of the background images of home and meeting pages were taken by Belem's family, they are licensed for use under Apache 2.0.
- Other images especially for "Change my background" feature are under the Unsplash license from [unsplash.com](https://unsplash.com/). 

> Unsplash license: All photos published on Unsplash can be used for free. You can use them for commercial and noncommercial purposes.

### CODE REVIEW

**Comment by ibelem** [comment]
@BruceDai please help to check why the Travis CI is failed, 

@huningxin @BruceDai @cuiyanx @miaobin Please review the code of WebNN Meeting example. I will integrate refactored BaseRunner.js when #1197 is fixed.

**Comment by BruceDai** [comment]
@ibelem Travis CI now uses Chrome/80.0.3987.149, we need upgrade chromedriver to be 80.0.2 to support Chrome/80.0.3987.149, I will submit PR to fix it, thanks.

**Comment by ibelem** [comment]
@huningxin @BruceDai @cuiyanx @miaobin Please review the code of WebNN Meeting example which integrated refactored BaseRunner.js with #1197 fix.

File: examples/webnnmeeting/assets/js/user/id.js [hunk 1]
```diff
@@ -327,27 +327,14 @@ export default {
...
-      return [scaledWidth, scaledHeight]
-    },
-    drawResultComponents(data, source) {
-      this.renderer.uploadNewTexture(source, this.getClippedSize(source))
-      this.renderer.drawOutputs(data)
-    },
     initRunner() {
       // eslint-disable-next-line new-cap
-      this.baserunner = new baseRunner(
+      this.baserunner = new BaseRunner(
```
Comment (BruceDai) [inline examples/webnnmeeting/assets/js/user/id.js:332]: @ibelem No need to create BaseRunner instance.

File: examples/webnnmeeting/assets/js/user/id.js [hunk 2]
```diff
@@ -327,27 +327,14 @@ export default {
...
         config.semanticsegmentation,
         this.updateProgress
       )
       // eslint-disable-next-line new-cap
-      this.runner = new semanticSegmentationRunner(
+      this.runner = new SemanticSegmentationRunner(
```
Comment (BruceDai) [inline examples/webnnmeeting/assets/js/user/id.js:337]: @ibelem This code needs update to 
```js
this.runner = new SemanticSegmentationRunner()
this.runner.setProgressHandler(this.updateProgress) 
```

**Comment by ibelem** [comment]
@BruceDai Thanks, I didn't this before but forgot to update it in previous commit, thanks, please review again.

**Comment by ibelem** [comment]
@BruceDai  10.239.47.52 means intranet ip not internet ip, so do we need to change this? Any company can have their internal ip like 10.239.47.52, am i correct? @huningxin

**Comment by huningxin** [comment]
> @BruceDai 10.239.47.52 means intranet ip not internet ip, so do we need to change this? Any company can have their internal ip like 10.239.47.52, am i correct? @huningxin

I would like to understand why we need a static IP?

**Comment by ibelem** [comment]
> > @BruceDai 10.239.47.52 means intranet ip not internet ip, so do we need to change this? Any company can have their internal ip like 10.239.47.52, am i correct? @huningxin
> 
> I would like to understand why we need a static IP?

It's the IP of the OWT WebRTC server which can be modified to any others.

**Comment by ibelem** [comment]
@BruceDai @huningxin I finally updated the static ip to xxx.xxx.xxx.xxx in config.js and readme, PTAL. With this change, our internal engineers must setup their own WebRTC server at first then they can test webnnmeeting front-end.

**Comment by huningxin** [comment]
> @BruceDai @huningxin I finally updated the static ip to xxx.xxx.xxx.xxx in config.js and readme, PTAL. With this change, our internal engineers must setup their own WebRTC server at first then they can test webnnmeeting front-end.

I think that's fine. Thanks @ibelem .

**Comment by BruceDai** [comment]
LGTM, thanks @ibelem

**Comment by ibelem** [comment]
Thanks all, merge this PR.

**Pull request merged**